qa/suites/rados/singleton-nomsgr/all/osd_stale_reads.yaml

```diff
@@ -0,0 +1,26 @@
+openstack:
+  - volumes: # attached to each instance
+      count: 3
+      size: 10 # GB
+roles:
+- [mon.a, mgr.x, osd.0, osd.1, osd.2, client.0]
+overrides:
+  ceph:
+    log-whitelist:
+      - \(OSD_DOWN\)
+      - \(POOL_APP_NOT_ENABLED\)
+      - \(SLOW_OPS\)
+      - \(PG_AVAILABILITY\)
+      - \(PG_DEGRADED\)
```

Comment:
http://pulpito.ceph.com/kchai-2019-09-29_02:16:32-rados-wip-kefu-testing-2019-09-28-1615-distro-basic-mira/4343377/

for posterity, the PG_DEGRADED failure was addressed by the change after this PR was picked up by my batch.